packages/template-retail-react-app/app/components/header/index.jsx

```diff
@@ -154,6 +125,11 @@ const Header = ({
         }
     }
 
+    const placeholder = intl.formatMessage({
```

Comment:
FYI the changes in this file are only for reverting an accessibility fix. For the release, we had to revert it for now.